network-api/networkapi/wagtailpages/templates/buyersguide/bg_base.html

```diff
@@ -90,7 +90,7 @@
           {% endif %}
           {% for cat in categories %}
             {% if cat.published_product_page_count > 0 %}
-              {% routablepageurl page 'category-view' cat.slug as cat_url %}
+              {% routablepageurl about_page 'category-view' cat.slug as cat_url %}
```

Comment:
`about_page` is the BuyersGuidePage. We had this in our context already and need this for routablepage urls.

Reply:
Done!